### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mrhouzlane**
option for macOs users

### CODE REVIEW

File: tutorials/README.md [hunk 1]
```diff
@@ -36,12 +36,34 @@ Some cryptographic functionalities of Aleph use curve secp256k1 and require inst
...
 and the [Python Aleph client](https://github.com/aleph-im/aleph-client) for it's command-line tools:
 
-    pip3 install uvicorn[standard] aleph-client fastapi eth_account
+- Linux : 
+ 
+``` 
+pip3 install uvicorn[standard] aleph-client fastapi eth_account
+``` 
+
+- macOs :
```
Comment (hoh) [inline tutorials/README.md:62]: Can you explain why different instructions for Linux and macOS when using pip3 ?

Comment (MHHukiewitz) [reply tutorials/README.md:62]: It works with quotes for Linux as well

Comment (hoh) [reply tutorials/README.md:62]: Then I do not understand why having different sections for Linux and a macOS, when they could be identical.

Comment (mrhouzlane) [reply tutorials/README.md:62]: should be one section but with quotes , that's the update

Comment (hoh) [reply tutorials/README.md:62]: These quotes also work on Linux. Can you update the document to use them for both OS ?

**Pull request merged**